lib/github.ts

```diff
@@ -651,6 +659,43 @@ export class FakeGitHub implements GitHub {
     return result;
   }
 
+  public forkRepository(repository: string | GitHubRepository, forkedRepositoryOwner: string): Promise<FakeGitHubRepository> {
+    return this.getRepository(repository)
+      .then((fakeRepository: FakeGitHubRepository) => {
```

Comment:
Why then instead of awaiting?

Reply:
I just got in the habit of doing it this way for `getRepository()` that I kept doing it. I don't think there's a good reason. I'll switch it over.

Reply:
I just changed a bunch of the `then()` calls to `async/await`.